.gitmodules

```diff
@@ -0,0 +1,16 @@
+[submodule "span-normalizer"]
+	path = span-normalizer
+	url = https://github.com/hypertrace/span-normalizer.git
+	branch = main
```

Comment:
I think its better to point to a released commit for better stability.

Reply:
I might be wrong but those modules are set in a commit unless we do git submodule update isn't? 
![image](https://user-images.githubusercontent.com/3075074/94272611-8e173a00-ff43-11ea-9d3e-f023b8b295f0.png)

Reply:
https://github.com/hypertrace/hypertrace-ingester/issues/8